src/MonoMod.UnitTest/MonoMod.UnitTest.csproj

```diff
@@ -11,12 +11,14 @@
 
     <TargetFrameworks>net46;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
     <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 16.6.0 And $(NETCoreAppMaximumVersion) &gt;= 5.0">$(TargetFrameworks);net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.0.0 And $(NETCoreAppMaximumVersion) &gt;= 6.0">$(TargetFrameworks);net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.0.0 And $(NETCoreAppMaximumVersion) &gt;= 6.0">net6.0;$(TargetFrameworks)</TargetFrameworks>
     <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.4.0 And $(NETCoreAppMaximumVersion) &gt;= 7.0">$(TargetFrameworks);net7.0</TargetFrameworks>
     <OutputType>Library</OutputType>
     <SignAssembly>skip</SignAssembly>
     <Nullable>annotations</Nullable>
 
+    <NoWarn>$(NoWarn);CS0649;CA1812</NoWarn>
```

Comment:
Both of these warnings are things that I'd rather be explicitly suppressed where they exist.

Reply:
Fixed - had to use pragmas for CS0649 though as it's a compiler warning.

Reply:
I believe you can `SuppressMessage` compiler built-in warnings, just not errors.